packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -10,29 +10,29 @@
  * governing permissions and limitations under the License.
  */
 
-import {ColumnProps, TableCollection} from '@react-types/table';
 import {GridNode} from '@react-types/grid';
 import {Key} from 'react';
 import {LayoutInfo, Point, Rect, Size} from '@react-stately/virtualizer';
 import {LayoutNode, ListLayout, ListLayoutOptions} from './ListLayout';
+import {TableCollection} from '@react-types/table';
 
 
 type TableLayoutOptions<T> = ListLayoutOptions<T> & {
-  getDefaultWidth: (props) => string | number
+  getColumnWidth: (key: Key) => number
 }
 
 export class TableLayout<T> extends ListLayout<T> {
   collection: TableCollection<T>;
   lastCollection: TableCollection<T>;
-  columnWidths: Map<Key, number>;
+  getColumnWidth_: (key: Key) => number;
```

Comment:
something public probably shouldn't have a trailing underscore, what's the reason for it?

Reply:
This is a good call. There was already a `getColumnWidth` used internally and when we were developing in here we didn't want to ruin the behavior because at the time it was being called externally. Now that we've swapped it out, i'll just swap the internal call to use the underscore and let the external call be without underscore 👍